indexer/cmd/indexer/cli.go

```diff
@@ -76,6 +76,18 @@ func runApi(ctx *cli.Context) error {
 	return api.Listen(apiCtx, cfg.API.Port)
 }
 
+func runAll(ctx *cli.Context) error {
```

Comment:
should we follow the pattern in `Indexer.Run()`. Where if either process errors out, it shuts down the other (by sharing a parent context that's cancelled).

Right now if the indexer stops running, the api server never halts. And if the API server crashes, then the indexer process isn't gracefully closed (which technically is okay)

Reply:
Nice! Yea definitely I wanted to do that and only didn't here because I didn't know how but see Indexer.Run() has an example of how

Reply:
Doing in a different pr since it will need a review that I did it right